libs/ui-lib/lib/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -112,25 +139,48 @@ export const isReservedIpv6Address = (ipv6Address: Address6) => {
   return ipv6Address.isLoopback() || ipv6Address.isMulticast();
 };
 
-export const isNotReservedHostIPAddress = (
-  protocolVersion: ProtocolVersion,
-  isDnsAddress = false,
-) => {
-  const textForAddress = isDnsAddress ? 'a DNS' : 'an interface';
+export const isNotReservedHostIPAddress = (protocolVersion?: ProtocolVersion) => {
+  return Yup.string().test(
+    'is-not-reserved-ip-address',
+    ({ value }) => {
+      const addresses = (value as string).split(',');
+      if (addresses.length === 1) {
+        return `Provided IP address is not a correct address for an interface.`;
+      }
+
+      const reservedAddresses = addresses.filter((address) => {
+        return isReservedAddress(address, protocolVersion);
+      });
+      return `Provided IP addresses ${reservedAddresses.join(
+        ', ',
+      )} are not correct addresses for an interface.`;
+    },
+    function (value) {
+      if (!value) {
+        return true;
+      }
+      // The field may admit multiple values as a comma-separated string
+      const addresses = (value as string).split(',');
+      return addresses.every((address) => !isReservedAddress(address, protocolVersion));
+    },
```

Comment:
Please favor arrow functions where possible. We should enforce this through ESLint...